lib/data_label.rb

```diff
@@ -0,0 +1,16 @@
+# a helper class to enumerate data section labels
+class DataLabel
+  @indices = Hash.new(0)
+
+  class << self
+    def indexed(label)
+      index = @indices[label]
+      @indices[label] += 1
+      "#{label}#{index}"
```

Comment:
could be one line, but would it clearer
```suggestion
      "#{label}#{@indices[label] += 1}"
```
The starting index would need to be `-1`